Framework/PythonInterface/test/python/plugins/algorithms/WorkflowAlgorithms/FitGaussianPeaksTest.py

```diff
@@ -514,6 +532,54 @@ def test_algorithm_does_not_throw_an_error_when_no_valid_peaks_fitted(self):
         self.assertEqual(mtd["peak_table"].rowCount(), 0)
         self.assertEqual(mtd["refit_peak_table"].rowCount(), 0)
 
+    def test_algorithm_uses_right_fit_window(self):
+        ws = CreateWorkspace([0, 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22] * 2,
+                             [0, 0, 0, 0, 1, 7, 0, 0, 0, 0, 10, 7] + [0] * 12, NSpec=2)
+
+        table = CreateEmptyTableWorkspace()
+        table.addColumn("float", "Centre")
+        table.addRow([20])
+        with mock.patch('plugins.algorithms.WorkflowAlgorithms.FitGaussianPeaks.FitGaussianPeaks.'
+                        'estimate_single_parameters') as mock_estimate_params:
+            mock_estimate_params.return_value = None
+            FitGaussianPeaks(InputWorkspace=ws,
+                             PeakGuessTable=table,
+                             EstimateFitWindow=False,
+                             FitWindowSize=11)
+            correct_x_val = np.array([0, 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22])
```

Comment:
these match the values in create ws. Shouldn't we define these first then use them in create ws?